src/posthog-core.ts

```diff
@@ -181,6 +184,8 @@ export const defaultConfig = (): PostHogConfig => ({
     session_idle_timeout_seconds: 30 * 60, // 30 minutes
     person_profiles: 'identified_only',
     __add_tracing_headers: false,
+    // by default the before capture fn is the identity function
```

Comment:
This comment makes no sense to me 😅

Reply:
Also why not make it optional and have nothing set? Feels clearer to me than having an instant return function

Reply:
where's your functional programming elitism?! the identity function just returns what you give it

it meant I didn't have to do any check to decide whether to call it, like we do iirc with `_onCapture` being a `__NOOP` fn